src/node_contextify.cc

```diff
@@ -956,27 +958,20 @@ class ContextifyScript : public BaseObject {
     bool timed_out = false;
     bool received_signal = false;
```

Comment:
Yeah sorry I meant to leave a note about that. The volatile ended up being unnecessary. The watchdogs take a non-const reference so that's good enough to ensure the optimizer doesn't do anything fishy. Then in the dtor `uv_thread_join` is called which will synchronize the watchdog thread back up with the main thread.